tests/unit/multitoken_library_test.cairo

```diff
@@ -0,0 +1,582 @@
+use openzeppelin::token::erc1155::interface::{ERC1155ABIDispatcher, ERC1155ABIDispatcherTrait};
+use openzeppelin::token::erc20::interface::{ERC20ABIDispatcher, ERC20ABIDispatcherTrait};
+use openzeppelin::token::erc721::interface::{ERC721ABIDispatcher, ERC721ABIDispatcherTrait};
+use pwn::mocks::{
+    erc20_mock::ERC20Mock, erc721_mock::ERC721Mock, erc1155_mock::ERC1155Mock,
+    account_mock::AccountMock
+};
+use pwn::multitoken::category_registry::MultiTokenCategoryRegistry;
+use snforge_std::{
+    declare, ContractClassTrait, store, load, map_entry_address, start_cheat_caller_address,
+    cheat_caller_address, CheatSpan, mock_call, start_mock_call, stop_mock_call
+};
+use starknet::ContractAddress;
+
+#[starknet::interface]
+trait IERC20Mock<TState> {
+    fn mint(ref self: TState, recipient: ContractAddress, amount: u256);
+}
+
+#[starknet::interface]
+trait IERC7210Mock<TState> {
+    fn mint(ref self: TState, recipient: ContractAddress, tokenId: u256);
+}
+
+#[starknet::interface]
+trait IERC1155Mock<TState> {
+    fn mint(ref self: TState, recipient: ContractAddress, tokenId: u256, value: u256);
+    fn batch_mint(
+        ref self: TState, recipient: ContractAddress, tokenIds: Span<u256>, values: Span<u256>
+    );
+}
+
+fn ALICE() -> ContractAddress {
+    starknet::contract_address_const::<'ACCOUNT2'>()
+}
+fn BOB() -> ContractAddress {
+    starknet::contract_address_const::<'ACCOUNT3'>()
+}
+
+fn deploy_accounts() -> (ContractAddress, ContractAddress) {
+    let contract = declare("AccountMock").unwrap();
```

Comment:
There are no actions to be performed by the account contract so alice and bob can just be contract_address_const

Reply:
btw leave the account mock file since we will need to test the signatures later on